### PR TITLE
implement logout functionality

### PR DESCRIPTION
**Pull request by trbck**
please ACK

### CODE REVIEW

**Comment by m619** [comment]
👍

File: zmon/controllers/NavigationVC.swift [hunk 1]
```diff
@@ -48,5 +48,20 @@ class NavigationVC: UINavigationController {
...
+        
+        //reset username and password
+        CredentialsStore.sharedInstance.clearCredentials()
+        //delete the token
+        CredentialsStore.sharedInstance.clearToken()
+        
+        //show the login view
+        
+        let result = self.storyboard?.instantiateViewControllerWithIdentifier("LoginVC")
+        self.presentViewController(result!, animated: false, completion: nil)
```
Comment (wojciechczerski) [inline zmon/controllers/NavigationVC.swift]: Please try changing the animated parameter to 'true'. Also there is the force unwrap operator:
`result!`
Try avoiding it.

Comment (trbck) [inline zmon/controllers/NavigationVC.swift]: changed the animated parameter to true. As discussed I'll leave the result! force unwrap operator.

**Comment by wojciechczerski** [comment]
👍

**Pull request merged**